SUMMARY.md

```diff
@@ -31,6 +31,10 @@
   * [Single-Machine Deployment](firehose-setup/cosmos/local-deployment.md)
 * [NEAR](firehose-setup/near/README.md)
   * [Single-Machine Deployment](firehose-setup/near/installation.md)
+* [Starknet](firehose-setup/starknet/README.md)
```

Comment:
Could you create a new section under `Firehose Setup` called `Community Integrations` and add the Starknet under it? We're trying to separate the chains we actually maintain and support.

I can also do it if you want to

Reply:
Sure lemme move everything over.